crates/bevy_asset/src/loader.rs

```diff
@@ -353,6 +353,11 @@ impl<'a> LoadContext<'a> {
 
     /// This will add the given `asset` as a "labeled [`Asset`]" with the `label` label.
     ///
+    /// Warning: this will not assign dependencies to the given `asset`. If adding an asset
```

Comment:
The warning word can either be capitalized or

Reply:
Looks important enough for correctness to be a heading, good catch.

Reply:
```suggestion
    /// # Warning
    ///
    /// This will not assign dependencies to the given `asset`. If adding an asset
```